recipes/libsimpleitk/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: libsimpleitk
+  version: "{{version}}"
+
+source:
+  git_url: git://itk.org/SimpleITK.git
+  git_tag:  v{{version}}
+  patches:
+    - fixes.patch
+
+build:
+    number: 0
+    skip: True   # [win] missing lua dependecy
```

Comment:
Please break out the comment from the selector. Perhaps putting the comment above the `skip` would be appropriate.